Main.java

```diff
@@ -9,7 +11,50 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    //create a Scanner for user input
+   Scanner input = new Scanner(System.in);
+
+  //ask user to chose an integer
+  System.out.println("Enter an integer");
+
+  //get integer from user
+  int integer1 = input.nextInt();
+  
+   //ask user to chose another integer
+  System.out.println("Enter another integer");
+
+  //get integer from user
+  int integer2 = input.nextInt();
+
+  //calculate the greater number
+  if(integer1 > integer2){
+  
+  //tell the user the greater number
+  System.out.println("The greater number is " + integer1);
+
```

Comment:
Good work but just watch the indentation. If it is inside an if statement, it should have an extra TAB over.